pkg/networkplugin-syncer/main.go

```diff
@@ -30,7 +30,13 @@ func main() {
 	// set up signals so we handle the first shutdown signal gracefully
 	stopCh := signals.SetupSignalHandler()
 
-	registry := event.NewRegistry("networkplugin-syncer", os.Getenv("NETWORK_PLUGIN"))
+	networkPlugin := os.Getenv("SUBMARINER_NETWORKPLUGIN")
+
+	if networkPlugin == "" {
+		networkPlugin = "generic"
```

Comment:
What does the "generic" plugin name refer to?  Should this be a constant somewhere?

Reply:
Yes, let's make constants for the network plugins on admiral on a 2nd pass, as for OVNKubernetes, etc...

Reply:
generic is means kubeproxy-iptables based, but not identified. It will trigger the kubeproxy driver as default